src/main/java/reactor/kafka/receiver/errors/LogAndFailReceiverExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
license header https://github.com/reactor/.github/blob/master/CONTRIBUTING.md#add-apache-license-header-to-all-new-classes